spec/namespaces/indices.yaml

```diff
@@ -3727,7 +3727,7 @@ components:
       description: Name of the index template to retrieve. Wildcard (*) expressions are supported.
       required: true
       schema:
-        $ref: '../schemas/_common.yaml#/components/schemas/Name'
+        $ref: '../schemas/_common.yaml#/components/schemas/Names'
       style: simple
```

Comment:
`get_index_template` doesn't actually support a comma separated list, this is something I corrected here: https://github.com/opensearch-project/opensearch-api-specification/commit/7a59510d30200b22792fb0bd5dd827b620030f89